packages/components/src/Emphasis/Emphasis.component.js

```diff
@@ -17,14 +17,18 @@ function emphasiseAll(text, value) {
 	if (!value) {
 		return text;
 	}
-	const strText = `${text}`;
-	const strValue = `${value}`;
 
-	return strText
-		.split(new RegExp(`(${escapeRegexCharacters(strValue)})`, 'gi'))
+	if (!Array.isArray(value)) {
+		value = [value];
+	}
+
+	const valuesUC = value.map(e => `${e}`.toLocaleUpperCase());
```

Comment:
```suggestion
	const valuesInUpperCase = value.map(e => `${e}`.toLocaleUpperCase());
```
Took me a while to understand UC 😬